explorer/src/index.css

```diff
@@ -0,0 +1,5 @@
+body {
```

Comment:
Also okay for now. In the future, I'd like to at least consider using [aphrodite] for styling. It's worked very well for me in the past. TL;DR: It combines the benefits of inline styling (locality of definitions, single-file components) with the full power of CSS—things like `:hover`, `::after`, `@keyframes`, and `@media` that you can't get with just `style` attributes.

(If you care about things like server rendering and minimizing bandwidth, it fares well there, too. But that's a discussion for another day.)

[aphrodite]: https://github.com/Khan/aphrodite